wp-content/themes/core/assets/js/src/admin/editor/components.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @module
+ * @exports init
+ * @description Initializes theme component JS in the block editor context.
+ */
+
+import * as tests from 'utils/tests';
+import * as tools from 'utils/tools';
+
+const init = () => {
+	// Note that initializing sliders in the block editor is incompatible with
+	// service workers because it requires data that's not available in the
+	// editor.
+	if ( ! tests.supportsWorkers() && tools.getNodes( '[data-js="c-slider"]', false, document, true )[ 0 ] ) {
```

Comment:
I SUGGEST removing `! tests.supportsWorkers()`.
The slider will still work even if `BLOCK_THEME_SERVICE_WORKER` is `false`.
```diff
- if ( ! tests.supportsWorkers() && tools.getNodes( '[data-js="c-slider"]', false, document, true )[ 0 ] ) {
+ if ( tools.getNodes( '[data-js="c-slider"]', false, document, true )[ 0 ] ) {
```